src/main/resources/com/puppycrawl/tools/checkstyle/checks/whitespace/messages_de.properties

```diff
@@ -18,3 +18,5 @@ ws.illegalFollow=Nach ''{0}'' folgt ein ungültiges Zeichen.
 ws.typeCast=Nach ''typumwandlung'' folgt kein Leerzeichen.
 
 empty.line.separator.multiple.lines = ''{0}'' verfügt über mehr als 1 Leerzeilen vor.
+
+single.space.separator=Verwenden Sie einen einzigen Raum ohne Leerzeichen zu trennen.
```

Comment:
translates to `Use a single room without space to separate.`. :joy_cat:

If it's useful for this PR, use something like `Verwenden Sie ein einfaches Leerzeichen um Zeichen zu trennen.`
[Translation from translate.google.com](https://translate.google.com/#de/en/Verwenden%20Sie%20ein%20einfaches%20Leerzeichen%20um%20Zeichen%20zu%20trennen)